mlir/lib/Dialect/Rock/Transforms/BlockwiseGemmToThreadwise.cpp

```diff
@@ -254,6 +276,13 @@ struct BlockwiseGemmRewritePattern
     toLdsIndexA.ignore("n_cuwave");
     toLdsIndexA.passThrough({"kpack"}, {2}, {"kpack"});
     TransformMapAttr toLdsIndexAAttr = toLdsIndexA.get();
+    SmallVector<Attribute, 4> transformAttrsA{splitTidAAttr, toLdsIndexAAttr};
+
+    // Since the LDS layout is `kOuter x rotate(m) x kpack` we want to rotate
+    // the dimension `m` before reading from LDS
+    int64_t strideA = (kPack == 1 ? copyMPerThread : 1);
+    rotateIf(op.getIsKContiguousDimA(), toLdsIndexA, toLdsIndexAAttr, strideA,
```

Comment:
Please put a cross-reference link to wrapLDSforStore to say that these two needs to be kept in sync.

TLDR;

I spent some time thinking about can we pass the transformMaps of the LDS so these need to not be maintained here.. it feels complex so lets keep a cross-reference here.

Reply:
The problem is also that on the accel path we don't have transform maps. I would pass the maps once we bring the accel path to use them.